### DISCUSSION TITLE
Thermostat Card Not Displaying Action

### DISCUSSION THREAD

**Discussion by dstewartwhite** [ScratMan/HASmartThermostat#216] [Q&A]
I have just moved from using ZHA to Zigbee2MQTT and the Thermostat Card is not displaying the action value, its always showing "Idle" even when its in "Heating" mode. With exactly the same code and parameters using ZHA the card either displays "Idle" or "Heating" whichever is relevant.
![image](https://github.com/ScratMan/HASmartThermostat/assets/114423605/a5c0189c-8a8b-4335-9adf-0679c2ebc9ee)
![image](https://github.com/ScratMan/HASmartThermostat/assets/114423605/5da2a65f-63dc-45b6-991f-4536b6879021)
Is there a setting that is required in Zigbee2MQTT that I am unaware of.
Many Thanks

**Comment by ScratMan** [comment]
Hi, please check the heater entity name is not different between ZHA and Z2M.

**Comment by dstewartwhite** [comment]
Many thanks for the quick reply. I took your suggestion and checked again the heater entity name and found that the copy and paste, I used to update it, had inserted a hidden white space. When corrected manually the Thermostat acted correctly

Dave

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

________________________________
